src/main/steps/task-list/template.njk

```diff
@@ -66,7 +66,7 @@
                                 <!-- First applicant - personal details -->
                                 <li class="app-task-list__item">
                                     <span class="app-task-list__task-name">
-                                        <a id="applicant1-personal-details" href="{{urls.APPLICANT_1_PERSONAL_DETAILS}}" class="govuk-link" aria-describedby="eligibility-completed">
+                                        <a id="applicant1-personal-details" href="{{urls.APPLICANT_1_FULL_NAME}}" class="govuk-link" aria-describedby="eligibility-completed">
                                             {{section2subheading2line1}}
                                         </a>
                                     </a>
```

Comment:
we also need logic to display correct status for personal details section, please refer to contact details

Reply:
Done